src/news.rs

```diff
@@ -163,6 +163,8 @@ impl News {
     }
 }
 
+// The warning is a false positive
```

Comment:
This is just me being slightly annoyed at the error so I silenced it ^^

Reply:
The fact that this makes CI fail makes me giggle slightly :D I guess I am locally on a newer version :)